.github/workflows/test-action.yml

```diff
@@ -65,3 +65,34 @@ jobs:
         else
           exit 0
         fi
+
+  test_ct_action_noverify:
+    runs-on: ubuntu-latest
+
+    name: Install chart-testing and test presence in path
```

Comment:
change this name to something more meaningful for this test

Reply:
Missed to update name after coping from basic test - updated the value.